tests/scripts/all.sh

```diff
@@ -439,6 +439,15 @@ msg "build: i386, make, gcc" # ~ 30s
 cleanup
 CC=gcc CFLAGS='-Werror -Wall -Wextra -m32' make
 
+msg "build: default config, MBEDTLS_RSA_NO_CRT, make, gcc"
```

Comment:
This comes in the middle of an "if x86_64" guarding 32/64 bit builds, which is not a correct place for this new test. Please move it to an appropriate place (preferably close to another test that also builds with some particular config set/unset, eg SHA1_IN_CERTS or TRUNCATED_HMAC or similar).